src/app/[locale]/specs/xrpc/page.mdx

```diff
@@ -53,7 +53,9 @@ There is also a legacy authentication scheme using HTTP Bearer auth with JWT tok
 
 Most requests should be authenticated using an access JWT, but the validity lifetime for these tokens is short. Every couple minutes, a new access JWT can be requested by hitting the `com.atproto.server.refreshSession` endpoint, using the refresh JWT instead of an access JWT.
 
-The JWTs themselves should be treated as opaque tokens.
+Clients should treat the tokens as opaque string tokens: the JWT fields and semantics are not a stable part of the specification.
+
+Servers (eg, PDS implementations) which generate and valiate auth JWTs should implement domain separation between access and refresh tokens, using the `typ` header field: access tokens should use `at+jwt`, and refresh tokens should use `refresh+jwt`.
```

Comment:
Just a note that `at+jwt` originates from [RFC 9068
](https://www.rfc-editor.org/rfc/rfc9068.html).  I mention it because I think it could be a common misconception that it's something we invented, i.e. that "at" comes from "AT" Protocol.

Reply:
Thanks! I didn't know that. updated text to clarify